hermes/probe/delete/delete_test.go

```diff
@@ -114,28 +114,25 @@ func genTestTarget(cfg *monitorpb.HermesProbeDef, t *testing.T) *probe.Target {
 	}
 }
 
-// setupMockSystem sets up the mock storage system through the fake client.
-func setupMockSystem(ctx context.Context, t *testing.T) stiface.Client {
+// setupFakeSystem sets up the fake storage system through the fake client.
+func setupFakeSystem(ctx context.Context, t *testing.T) stiface.Client {
```

Comment:
Nit: I would call this `fakeStorageClient`.
Then the call-site reads better:
```
client := fakeStorageClient(...)
```

Reply:
Ack.